kafka_consumer/metadata.csv

```diff
@@ -2,3 +2,4 @@ metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation
 kafka.broker_offset,gauge,,offset,,Current message offset on broker.,0,kafka,broker offset,
 kafka.consumer_lag,gauge,,offset,,Lag in messages between consumer and broker.,-1,kafka,consumer lag,
 kafka.consumer_offset,gauge,,offset,,Current message offset on consumer.,0,kafka,consumer offset,
+kafka.consumer_lag_seconds,gauge,,seconds,,Lag in seconds between consumer and broker.,-1,kafka,consumer time lag,
```

Comment:
```suggestion
kafka.consumer_lag_seconds,gauge,,second,,Lag in seconds between consumer and broker.,-1,kafka,consumer time lag,
```
This is failing validation because the unit needs to be singular